hivemq-edge/src/frontend/src/extensions/datahub/locales/en/datahub.json

```diff
@@ -269,11 +269,23 @@
       "topicFilter": {
         "duplicate": "The topic filter {{ filter }} is already defined",
         "alreadyMatching": "A policy is already matching the topic filter {{ filter }}"
+      },
+      "dataPolicy": {
+        "notLoading": "Cannot load the existing policies for checking ids",
+        "notUnique": "The id is already in use by another data policy"
+      },
+      "behaviourPolicy": {
+        "notLoading": "Cannot load the existing policies for checking ids",
+        "notUnique": "The id is already in use by another behaviour policy"
```

Comment:
nit: In Cloud, we Americanize by default (=> `behaviour` -> `behavior`). I do prefer British English too but hey 🤷‍♂️. Is there such a rule in Edge and DataHub?
Should there be one at the guild level @h2xd?

Reply:
Very good question that @RobinAtherton also raised between `Organisation` and `Organization`. 
I believe that it would be nice that we have a recommendation on the guild level for now. I add in a point for the next agenda.

Reply:
In my opinion, given that the website is `en-us`, we should stick to it in products as well. However, with `i18n` supposedly used everywhere, it would be super easy to create a `en-gb` version for the few key/values that differ. Whatever key isn't present in `en-gb` or other languages will revert to the default anyway